File: app/models/helper/BelgacomChannel.java
package models.helper;

import java.util.List;


import com.google.gson.annotations.SerializedName;

public class BelgacomChannel {
	
	@SerializedName("cid")
	public String channelId;
	
	@SerializedName("cname")
	public String channelname;
	
	@SerializedName("cnum")
	public String channelNumber;
	
	@SerializedName("clogo")
	public String channelLogo;
	
	public List<BelgacomProgram> pr;
	
	//public String liveid; // no idea
	//public String offset; // visualisation start offset in pixels?
	//public int idx; // visualisation index
	//public String fill; // visualisation fill?
}

File: app/models/helper/BelgacomChannel.scala
package models.helper

case class BelgacomChannel(
  cid:String,
  cname:String,
	cnum:String,
	clogo:String,
	pr:List[BelgacomProgram]    
) {

  val channelId = cid
  val channelName = cname
  val channelNumber = cnum
  val channelLogo = clogo

  override def toString = s"$channelName [${pr.size}]"
}